ivy_tests/test_ivy/test_frontends/test_torch/test_pooling_functions.py

```diff
@@ -269,3 +269,42 @@ def test_torch_lp_pool1d(
         stride=stride[0],
         ceil_mode=False,
     )
+
+
+# avg_pool2d
+@handle_frontend_test(
+    fn_tree="torch.nn.functional.lp_pool2d",
+    dtype_x_k_s=helpers.arrays_for_pooling(
+        min_dims=4,
+        max_dims=4,
+        min_side=1,
+        max_side=4,
+    ),
+    norm_type=helpers.number(min_value=0.1, max_value=6),
+    test_with_out=st.just(False),
+)
+def test_torch_lp_pool2d(
+    dtype_x_k_s,
+    norm_type,
+    *,
+    test_flags,
+    frontend,
+    fn_tree,
+    on_device,
+):
+    input_dtype, x, kernel_size, stride, _ = dtype_x_k_s
+    # Torch ground truth func expects input to be consistent
+    # with a channels first format i.e. NCW
+    x[0] = x[0].reshape((x[0].shape[0], x[0].shape[-1], *x[0].shape[1:-1]))
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        test_flags=test_flags,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=x[0],
+        norm_type=norm_type if norm_type > 0 else 1,
```

Comment:
this check does not seem to be necessary as the `norm_type` has a minimum value of 0.1 so it is never going to be below 0

Reply:
I agree, but in practice I tried testing without it and I saw that even with `min_value` set, `norm_type` still sometimes has value 0. I don't know why, maybe a bug in `helpers.number`?

Reply:
How do I proceed here?

Reply:
I just checked and you are right, It's all good on your side, I'll merge and report this behavior to the team.